deployment/copilot/kafka-puller/addons/taskRole.yml

```diff
@@ -0,0 +1,43 @@
+# You can use any of these parameters to create conditions or mappings in your template.
```

Comment:
Can you briefly describe addons in the README for copilot?
Are you using this?  I thought that this was being done in the CDK